### ISSUE TITLE
.generate() does not work on crk-normative-generator.fomabin

### ISSUE THREAD

**Issue by Madoshakalaka** [eddieantonio/fst-lookup#10]
I'm using the latest cree fomabin
```python
fst = FST('crk-normative-generator.fomabin')
list(fst.generate('nîskâw+V+II+Cnj+Prs+3Sg'))
```
```python
>>> []
```

```python
fst = FST('crk-normative-generator.fomabin')
list(fst.analyze('nîskâw+V+II+Cnj+Prs+3Sg'))
```
```python
>>> [('nîskâk',)]
```

I'm confused:
`generate` should work, `analyze` shouldn't, but it's the other way around here.

Also a small inconsistency: `readme.md` mentions word generation returns type Generator[str], while right here it gives Generator[Tuple[str]]

**Comment by Madoshakalaka** [comment]
matter of fact, in the old Cree intelligent dictionary app, students have been using only `FST.analyze` for both generation and analysis. That what I did later too.

**Comment by Madoshakalaka** [comment]
summary: `FST.generate` doesn't work with cree fomabins, `FST.analyze` works for both generation and analysis

**Comment by eddieantonio** [comment]
I believe this is a genuine bug; however, I'll likely address it on Friday!

**Comment by eddieantonio** [comment]
To be clear, the _expected_ behaviour is this:

```python
>>> fst = FST('crk-normative-generator.fomabin')
>>> list(fst.generate('nîskâw+V+II+Cnj+Prs+3Sg'))
['nîskâk']
```

**Comment by eddieantonio** [comment]
Also, @Madoshakalaka please edit the issue title to be more descriptive of the actual bug! It helps when searching for closed issues and what-not.

**Comment by eddieantonio** [comment]
This has been fixed as of v2019.7.17 (d7da53b5cb2f3b4dcc06804e90551d66f9d4965f)/

The correct way to do this is to **invert the FST** on load. Here is a minimal working example:

```python
from fst_lookup import FST

fst = FST.from_file('crk-normative-generator.fomabin', labels='invert')
print(*fst.generate('nîskâw+V+II+Cnj+Prs+3Sg'))
```

Which prints "nîskâk", as expected.

**Comment by Madoshakalaka** [comment]
wow using the unpack operator is so smart
